tests/devicelibs_test/btrfs_test.py

```diff
@@ -164,19 +168,5 @@ def testSubvolume(self):
         subvolumes = btrfs.list_subvolumes(self.mountpoint)
         self.assertEqual(len([v for v in subvolumes if v['path'].find("SV1.1") != -1]), 1)
 
-class BTRFSAsRootTestCase3(loopbackedtestcase.LoopBackedTestCase):
-
-    def __init__(self, methodName='runTest'):
-        super(BTRFSAsRootTestCase3, self).__init__(methodName=methodName, deviceSpec=[8192])
-
-    def testSmallDevice(self):
-        """ Creation of a smallish device will result in an error if the
-            data and metadata levels are specified differently, but not if
-            they are unspecified.
-        """
-        with self.assertRaises(BTRFSError):
-            btrfs.create_volume(self.loopDevices, data="single", metadata="dup")
-        self.assertEqual(btrfs.create_volume(self.loopDevices), 0)
-
```

Comment:
I played with it for a bit and couldn't find a size that was right on the line like the test expects and btrfs is not worth the time it would take at this point.